src/main/java/com/manardenza/dao/UserDaoImpl.java

```diff
@@ -18,6 +19,11 @@ public static UserDaoImpl getInstance() {
         return instance;
     }
 
+    public User getUserByName(String firstName, String lastName) {
+        return database.stream().filter(param -> param.getFirstName().equals(firstName) && param.getLastName()
```

Comment:
Use additional .filter() statement instead of && - this will make code more readable